spec/features/account_reset/delete_account_spec.rb

```diff
@@ -173,4 +185,92 @@
       expect(page).to have_current_path(new_user_session_path)
     end
   end
+
+  context 'logs IRS attempts api events' do
+    before do
+      allow(IdentityConfig.store).to receive(:irs_attempt_api_enabled).and_return(true)
+      mock_irs_attempts_api_encryption_key
+    end
+
+    it 'allows the user to delete their account after 24 hours and log irs event' do
+      visit_idp_from_ial1_oidc_sp(
+        client_id: service_provider.issuer,
+      )
+
+      signin(user_email, user.password)
+      click_link t('two_factor_authentication.login_options_link_text')
+      click_link t('two_factor_authentication.account_reset.link')
+      expect(page).
+        to have_content strip_tags(
+          t('account_reset.recovery_options.try_method_again'),
+        )
+      click_link t('account_reset.request.yes_continue')
+      expect(page).
+        to have_content strip_tags(
+          t('account_reset.request.delete_account'),
+        )
+      click_button t('account_reset.request.yes_continue')
+
+      expect(page).
+        to have_content strip_tags(
+          t('account_reset.confirm_request.instructions_start'),
+        )
+      expect(page).
+        to have_content user_email
+      expect(page).
+        to have_content strip_tags(
+          t('account_reset.confirm_request.instructions_end'),
+        )
+      expect(page).to have_content t('account_reset.confirm_request.security_note')
+      expect(page).to have_content t('account_reset.confirm_request.close_window')
+
+      reset_email
+      set_new_browser_session
+      events = irs_attempts_api_tracked_events(timestamp: Time.zone.now)
+      expected_event_types = %w[mfa-login-phone-otp-sent login-email-and-password-auth
+                                account-reset-request-submitted]
+      received_event_types = events.map(&:event_type)
+
+      expect(events.count).to eq received_event_types.count
+      expect(received_event_types).to match_array(expected_event_types)
+
+      travel_to(Time.zone.now + 2.days + 1) do
```

Comment:
I would add a unit to make this clear
```suggestion
      travel_to(Time.zone.now + 2.days + 1.second) do
```